GOVERNANCE.md

```diff
@@ -122,3 +122,10 @@ If an agenda item cannot reach a consensus, a CTC member can call for either a
 closing vote or a vote to table the issue to the next meeting. The call for a
 vote must be approved by a simple majority of the CTC or else the discussion
 will continue.
+
+A vote may be scheduled for a future date to allow time for CTC members to
+better educate themselves about relevant issues. A vote is not valid unless
+there is participation of a quorum of the CTC. A CTC quorum is more than half of
+the CTC. Absentee votes may be cast in advance by commenting on the weekly CTC
```

Comment:
@Fishrock wrote:

> This is incorrect.

> It needs to be half of the entire CTC.

The "needs to be half of the entire CTC" text in the current document refers specifically to *calls* to vote (that is, a *motion* to close) on a contentious subject, not the votes on the contentious subjects themselves. (Or at least that's how it appears to me.)

And there's a good reason for that. If those were the rules, then:

* It only takes a small number of inactive members or abstentions to make it impossible to take action on a divisive issue.
* It puts pressure on people who maybe should abstain to actually cast an ill-informed vote instead. If we're making a decision that only half the body is adequately informed and has adequate expertise to address, then just that half should have a say. People who know what they don't know should be able to abstain without making gridlock more likely. I don't think it's wise or realistic to expect people on the CTC (or anywhere in the project) to understand everything about the project. Abstaining is OK as long as you're not making it a way of life on every topic.
* This makes proposals easy to block (basically, anyone who abstains or doesn't participate is essentially a "no" vote). In itself, that may not be a bad thing, but this encourages gaming the system. If you want your way, propose the *opposite* of what you want as the text to be voted on and you've got a head start because anyone not voting is effectively voting for your side. This is not a problem on a motion to close discussion because you can't have a motion to *not* close. There's a motion to close and a motion to table and that's it. But it is conceivably a problem when it comes to actual voting on the contentious issue at hand.

To my reading, at least, the current doc does not address how we handle votes on contentious issues&mdash;only how we handle the votes on motions to close and motions to table. I'm not sure anyone really has been working with a static and fully-formed idea of how these votes work. The new text is an attempt to document what I believe is a workable framework. We haven't needed it, really, so far. As I've noted, votes tend to be lopsided. But we might need it one day and it will be best to have an explicit and unambiguous process. (This text all really only becomes relevant when the Consensus-Seeking has failed.)